Dockerfile

```diff
@@ -6,11 +6,10 @@ RUN apt-get update && apt-get install -y \
     curl \
     unzip && \
   cd /opt/ && \
-  #curl -s https://api.github.com/repos/balena-io/balena-cli/releases/latest | \
-  #  grep browser_download_url.*balena-cli-v.*-linux-x64-standalone.zip | \
-  #  cut -d : -f 2,3 | \
-  #  xargs -n 1 curl -O -sSL && \
-  curl -O -sSL https://github.com/balena-io/balena-cli/releases/download/v11.31.17/balena-cli-v11.31.17-linux-x64-standalone.zip && \
```

Comment:
Some to be aware though: there are breaking changes between balena-cli v11 and v12, as per release notes:
https://github.com/balena-io/balena-cli/wiki/CLI-v12-Release-Notes